CREDITS.md

```diff
@@ -0,0 +1,12 @@
+## Libraries
+
+The following software libraries require a dedicated copyright notice for distribution.
+
+#### MPL-2.0
+
+- **MPL-2.0** &lt;<http://crates.io/crates/colored>&gt;
+- **MPL-2.0** &lt;<http://crates.io/crates/indent>&gt;
+
+#### BSD-3-Clause
+
+- **BSD-3-Clause** &lt;<http://crates.io/crates/subtle>&gt;
```

Comment:
This seems really strangely formatted. You have a section header for the license and then bold the license ahead of each library. This is redundant and you just need to mention the correct license once per library.